### ISSUE TITLE
Unable to View TODO in toggle

### ISSUE THREAD

**Issue by prateek531990** [reergymerej/todo#37]
Syntax tried in .py file

# TODO: test1
// TODO: test2
/ TODO: test3

No results in toggle menu

**Comment by reergymerej** [comment]
Thanks, @prateek531990.  I'll check it out.

**Comment by reergymerej** [comment]
```py
# TODO: Add world
// TODO: test2
/ TODO: test3
print("Hello")
```
This seems to work for me.  Did you change the `RegExp Pattern` perhaps?

![](https://i.imgur.com/081fCuE.png)

**Comment by prateek531990** [comment]
RegExp Patterns are default,
please have a look
![capture](https://user-images.githubusercontent.com/19922603/37267650-b4aa4474-25e7-11e8-8dd2-69aea3da7bb6.JPG)
![test](https://user-images.githubusercontent.com/19922603/37267652-b93d57a6-25e7-11e8-89c7-4989c24cbfb0.JPG)

**Comment by prateek531990** [comment]
Perhaps it is not searching for it on all open files/ directory, 3  scripts are open and it showing searched 2 files?

**Comment by reergymerej** [comment]
You're right.  It's searching the workspace, but I guess that doesn't include _other_ files in the editor.

https://github.com/reergymerej/todo/blob/master/lib/service.js#L58
https://github.com/atom/atom/blob/v1.24.1/src/workspace.js#L1805

![](https://i.imgur.com/4lda2Ec.png)

Thanks for reporting this.  Any interest in submitting a PR?

**Comment by jonatjano** [comment]
My forks should do that after I find why what I've currently done break the thing ^^, I'll do a PR when it will solve #38

**Comment by jonatjano** [comment]
@reergymerej shouldn't this issue be renamed to something like "TODOs not found in file opened but not saved in workspace"
